config/gtec_publisher.yaml

```
@@ -0,0 +1,18 @@
+gtec_eeg_publisher:
+  ros__parameters:
+    num_channels: 8 # Number of EEG channels to acquire
+    num_samples: 32 # Number of samples per data block/message
+    sampling_rate: 256.0 # Sampling frequency in Hz
```

I checked the manual:
```suggestion
    sampling_rate: 256.0 # Sampling frequency in Hz. Valid Values (32, 64, 128, 256, 512, 600, 1200, 2400, 4800)
```